src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/index.ts

```diff
@@ -132,6 +132,16 @@ export interface IFormRadioButtonComponent extends IFormComponent {
 
 export interface IFormTextAreaComponent extends IFormComponent { }
 
+export interface ILayoutSets {
```

Comment:
Do we need this interface? Could we instead use the type `ILayoutSet[]` directly in the reducer? Then we could avoid this extra level of nesting in the state.

Reply:
I keep it in case we want some additional parameters later for sets.